packages/classic/src/component/Component.ts

```diff
@@ -137,16 +137,42 @@ const recursivelyInherit = (world: World, baseEid: number, inheritedEid: number
  * @param {number} eid
  * @param {boolean} [reset=false]
  */
-export const addComponent = (world: World, component: Component, eid: number, reset = false) => {
+export const addComponent = (world: World, eid: number, component: Component, reset = false) => {
```

Comment:
I like this parameter order a lot better, but I want to leave `addComponent` as-is for backwards compatibility reasons and introduce an alternative API that I feel is more ergonomic:
```ts
import { Component } from 'bitecs'
Component.add(world, Position, eid) 
```